spk/firebird/Makefile

```diff
@@ -0,0 +1,33 @@
+SPK_NAME = firebird
+SPK_VERS = 4.0.3
+SPK_ICON = src/firebird.png
+SPK_REV = 1
+
+BETA = 1
+
```

Comment:
Beta packages will not be displayed in the package center on DSM >= 7 (beta is supported only by official packages by Synology).

So probably we should remove this (and firebird itself seems to be stable).

Reply:
Done.